lib/streamlit/elements/media.py

```diff
@@ -52,13 +55,22 @@ def audio(
         data : str, bytes, BytesIO, numpy.ndarray, or file opened with
                 io.open().
             Raw audio data, filename, or a URL pointing to the file to load.
-            Numpy arrays and raw data formats must include all necessary file
+            Raw data formats must include all necessary file
             headers to match specified file format.
+            In case of numpy array:
```

Comment:
If `data` is a numpy array, it must either be a 1D array of the waveform or a 2D array of shape `(num_channels, num_samples)` with waveforms for all channels. See the default channel order at http://msdn.microsoft.com/en-us/library/windows/hardware/dn653308(v=vs.85).aspx